docs/kusama-ledger.md

```diff
@@ -0,0 +1,170 @@
+
+This is a list of what you will need before starting:
+
+- A Ledger Nano S or a Ledger Nano X.
+- The latest firmware installed. (At the time of writing this was 1.6)
+- Ledger Live is installed.
+- A web browser is installed that you can use to access [Polkadot-JS Apps][].
+
+## Installing the Ledger Application
+
+### Using Ledger Live
+
+> NOTE: If you are reading this notice the Kusama application has not been approved for Ledger Live
+> yet. If you would like to proceed with the pre-release installation please follow the "Using the
+> Developer Release" instruction below. Otherwise hold tight, the application should be approved
+> soon!
+
+- Open the "Manager" tab in Ledger Live.
+- Connect and unlock your Ledger device.
+- If asked, allow the manager on your device by pressing the right button.
+- Find Kusama in the app catalog.
+- Click the "Install" button of the app.
+- An installation window appears.
+- Your device will display "Processing…"
+- The app installation is confirmed.
+
+Please proceed to the usage instructions below.
+
+### Using the Developer Release
+
+> NOTE: These instructions are for development installation only. It is recommended to install the
+> application from Ledger Live unless you _know exactly what you're doing_.
+
+Instructions for downloading the prerelease binary from the GitHub releases is written [on the
+README][prerelease instructions] for the Kusama Ledger application GitHub repository.
+
+On the [releases page][] you can download the shell script `install_app.sh` and then make it
+executable in your shell by typing the command `chmod +x install_app.sh`.
+
+Using `install_app.sh` help command will show you the available options:
+
+```zsh
+$ ./install_app.sh --help
+Zondax Installer [Kusama-1.2011.1] [Warning: use only for test/demo apps]
+  load    - Load Kusama app
+  delete  - Delete Kusama app
+  version - Show Kusama app version
+```
+
+Next, you must make sure your Ledger device is plugged in and unlocked and you're using the latest
+firmware (1.6 at the time of writing). If everything is prepared, then type `install_app.sh load`
+and accept the prompts on your Ledger device to install the application.
+
+First it will prompt you to allow an unsafe manager. Navigate all the way to the left and click both
+buttons to allow the unsafe manager to install the Kusama app.
+
+After some processing time, the screen of your device will update to say "Install app Kusama".
+Again, navigate all the way to the right, verify the Identifier hash matches the one that is printed
+in your terminal. Click both buttons on "Perform Installation" to install the application. It will
+ask again for your Pin code and you should enter it in the device.
+
+At the end of the process you should the newly installed Kusama application on the device. Great
+job!
+
+## Using on Polkadot-JS Apps
+
+### Loading Your Account
+
+[Polkadot-JS Apps][] already has an integration with the Ledger application so that your device will
```

Comment:
We'll have to talk to Jaco about turning this off at some point and edit the wiki.